src/features/welcome/welcome-page.js

```diff
@@ -1,17 +1,181 @@
+            imgPath={image2}
+            imgAlt=""
+          />
+          <Feature
+            title="Track your payment status"
+            text="No need to worry about forget getting paid. Billy helps you keep on eye on pending and overdue payments. Let’s get paid faster!"
+            imgPath={image3}
+            imgAlt=""
+          />
+          <Feature
+            title="Manage business income and expense"
+            text="Billy easily manages all your projects, customers and invoices in one place. Get report of your business income and expense anywhere."
+            imgPath={image4}
+            imgAlt=""
+          />
+        </section>
+        <section className="madeSimple">
+          <h2 className="titleBlack titleCenter">Invoicing made simple</h2>
+          <Button variant="contained" className="block buttonCenter">Try now for FREE</Button>
+        </section>
+        <section className="team">
+          <h2 className="titleBlack">Our Team</h2>
+          <TeamMember
+            memberName="Gabriel Gimenes"
+            memberImage="https://picsum.photos/200?random=4"
+            memberImageAlt=""
+            memberPosition="Project Manager / Developer"
+            memberLinkedin=""
+          />
+          <TeamMember
+            memberName="Ashim Karki"
+            memberImage="https://picsum.photos/200?random=5"
+            memberImageAlt=""
+            memberPosition="Lead Developer"
+            memberLinkedin=""
+          />
+          <TeamMember
+            memberName="Luisa Rueda"
+            memberImage="https://picsum.photos/200?random=6"
+            memberImageAlt=""
+            memberPosition="Developer"
+            memberLinkedin=""
+          />
+          <TeamMember
+            memberName="Karen Garcia"
+            memberImage="https://picsum.photos/200?random=7"
+            memberImageAlt=""
+            memberPosition="Developer"
+            memberLinkedin=""
+          />
+          <TeamMember
+            memberName="Mita Trisnodjojo"
+            memberImage="https://picsum.photos/200?random=8"
+            memberImageAlt=""
+            memberPosition="Lead Designer"
+            memberLinkedin=""
+          />
+          <TeamMember
+            memberName="Karen Chiu"
+            memberImage="https://picsum.photos/200?random=9"
+            memberImageAlt=""
+            memberPosition="Designer"
+            memberLinkedin=""
+          />
+          <TeamMember
+            memberName="Dexter Bolasoc"
+            memberImage="https://picsum.photos/200?random=10"
+            memberImageAlt=""
+            memberPosition="Designer"
+            memberLinkedin=""
+          />
+          <TeamMember
+            memberName="Milly Tsou"
+            memberImage="https://picsum.photos/200?random=11"
+            memberImageAlt=""
+            memberPosition="Designer"
+            memberLinkedin=""
+          />
```

Comment:
It is better to have all these team members in an array and then iterate it instead of duplicating over and over the component.